authors/apps/articles/renderers.py

```diff
@@ -4,10 +4,10 @@
 from rest_framework.utils.serializer_helpers import ReturnList
 
 
-class ArticleJSONRenderer(JSONRenderer):
+class ArticleJSONRenderer(JSONRenderer):# NOQA
```

Comment:
All this isn't on your task. Let me know where you think it is originating from.

Reply:
these are from Moses tasks, I rebased from his task

Reply:
follow this article on what you should do now.
https://softwareengineering.stackexchange.com/questions/351727/working-on-a-branch-with-a-dependence-on-another-branch-that-is-being-reviewed

Reply:
Noted, thanks

Reply:
did you do as the article said??

Reply:
I believe it's fixed now